### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by anas599**
In this PR:

- Create a new database based on a diagram
- add many to many and many to one for tables
- add foreign key and constraint

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  status TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  contact_info VARCHAR(255)
+);
+
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql:17]: Good work creating the tables

- Kindly create the table missing for `invoices`, in this way you will be following the schema provided on the project 💯

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ALTER TABLE medical_histories
+ADD CONSTRAINT fk_medical_history_id
+FOREIGN KEY (medical_history_id)
+REFERENCES medical_histories(id);
+
+CREATE TABLE medical_treatment(
+  medical_histories_id INTEGER REFERENCES medical_histories (id),
+  treatments_id INTEGER REFERENCES treatments(id),
+);
```
Comment (BregornOriginal) [inline schema_based_on_diagram.sql]: - Kindly create INDEX for each table as it's required on the project requirements

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @anas599  

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more


## Highlights

✔️ Linters are passing
✔️ Good commit history
✔️ Good Pull Request description
✔️ README looks professional 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**